File: programme/index-of-alphabet/index-of-alphabet.py
# Write a programme to find index of an alphabet

def index_of_alphabet(letter):
    alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l',
                'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
    letter = letter.lower()
    return alphabet.index(letter)+1


print("Input  :", 'a')
print("Output :", index_of_alphabet('a'))

File: programme/index-of-alphabet/indexofalphabet.java
// Write a programme to find index of an alphabet
import java.util.*;

class IndexOfAlphabet {

  public static void main(String args[]) {
    System.out.print("Input  : ");
    Scanner sc = new Scanner(System.in);
    char c = sc.next().charAt(0);
    if (Character.isLowerCase(c)) {
      System.out.println("Output : " + (c - 96));
    } else {
      System.out.println("Output : " + (c - 64));
    }

    sc.close();
  }
}
